shipyard_airflow/dags/openstack_api_call.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should probably start our own dags/samples directory as this isn't really a "real" dag?